recipes/keras/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "1.0.7" %}
+{% set name = "keras" %}
+{% set sha256 = "b94158c9dd09d3af79acf0ea06ed0084d4645b0ea1b62168f10929260731b14e" %}
+{% set github_owner = "fchollet" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/{{ github_owner }}/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
I feel if we go this way `name` is not guaranteed to be the repo name. So it should either have a different jinja variable or this extra templating (including owner, not version) should be dropped.